docs/src/pages/theming/theming.web.mdx

```diff
@@ -106,6 +106,45 @@ const myTheme = {
 };
 ```
 
+#### Component token definitions
+
+Amplify UI follows a consistent pattern when defining tokens for a component's states and variations. This is helpful for discovering what tokens are available for theming different aspects of a component. Amplify UI uses the following pattern:
+
+```javascript
+component[variant][_state][child];
```

Comment:
Should we change `[variant]` to `[modifier]`? Some components have a `variant` prop as well as a `size` prop

Reply:
That works for me; maybe a little more inclusive of both variants (primary, link, etc.) and size.

Reply:
Updated to include this change, and added some definitions for those terms (modifier, state).